packages/application-components/src/components/dialogs/info-dialog/README.md

```diff
@@ -0,0 +1,41 @@
+# InfoDialog
+
+## Usage
+
+```js
+import { InfoDialog } from '@commercetools-frontend/application-components';
+```
+
+#### Description
+
+Info dialogs are controlled components used to show more information about a particular feature, UI element, etc. They are mainly composed of text paragraphs.
+
+#### Usage
+
+```js
+<InfoDialog
+  title={<FormattedMessage {...messages.infoTitle} />}
+  isOpen={isOpen}
+  onClose={handleClose}
+>
+  <Spacings.Stack scale="m">
+    <div>
+      <Text.Body>{'Lorem ipsus ...'}</Text.Body>
+    </div>
+    <div>
+      <Text.Body>{'Lorem ipsus ...'}</Text.Body>
+    </div>
+  </Spacings.Stack>
+</InfoDialog>
+```
+
+#### Properties
+
+| Props                  | Type       | Required | Values   | Default | Description                                                                                                                 |
+| ---------------------- | ---------- | :------: | -------- | ------- | --------------------------------------------------------------------------------------------------------------------------- |
+| `title`                | `node`     |    ✅    | -        | -       | The title of the Info Dialog                                                                                                |
```

Comment:
is this a node to make it easier to pass in `react-intls` formatted message? Can we make it a `string`?

Reply:
I thought about it and decided to keep the API open for now. In theory, we should be able to pass a string, but I'm not sure if we might need to pass a react component from time to time.
I can double check though. For me it's also fine to restrict it to a string.